vmm/src/device_manager.rs

```
@@ -2350,6 +2350,9 @@ impl DeviceManager {
             ConsoleOutput::Socket(_) => {
                 return Err(DeviceManagerError::NoSocketOptionSupportForConsoleDevice);
             }
+            ConsoleOutput::Tcp(_) => {
```

nit: I know the other code paths doesn't do it as well but I think it would be nice to have. Feel free to ignore. The idea here is to not miss any hidden context of the underlying error

```
ConsoleOutput::Tcp(e) = {
   log::error!("tcp console output error: {e}");
   return Err() 
}

---

This code path doesn't indicate an error in the `Tcp` value, but rather we throw an error here because we do not support Tcp mode for the `console` configuration, but only `serial`.

So no underlying error to miss here I think